src/matlab2tikz.m

```diff
@@ -6329,6 +6327,14 @@ function printAll(m2t, env, fid)
     end
     opts = opts_append(opts, key, value);
 end
+function opts = opts_addOpts(m2t, opts, key, subOpts, sep)
```

Comment:
I thought the same (except for the `opts_addSubOpts`), but the name change is fine with me. Not passing `sep` has been implemented in the meanwhile.